pkg/buildsettings/buildSettings.go

```diff
@@ -0,0 +1,101 @@
+package buildsettings
+
+import (
+	"encoding/json"
+	"reflect"
+
+	"github.com/SAP/jenkins-library/pkg/log"
+	"github.com/pkg/errors"
+)
+
+type BuildSettingsInfo struct {
```

Comment:
as a second thought , do we need do we need this struct  `BuildSettingsInfo `? , when the step passes the buildSettingsInfo we can accept it as a `string `and since we any way unmarshal it , it should not be a problem,